node/globals.ts

```diff
@@ -163,6 +163,7 @@ declare global {
     page: number
     pageSize: number
     where: string
+    schema: string
```

Comment:
schema can be undefined, right ?
```suggestion
    schema?: string
```